core/src/main/java/com/yahoo/oak/EntryArrayDirect.java

```diff
@@ -70,25 +70,25 @@ public int fieldCount() {
     /** {@inheritDoc} */
     @Override
     public void clear() {
-        UnsafeUtils.UNSAFE.setMemory(null, entriesAddress, buffer.capacity(), (byte) 0);
+        DirectUtils.UNSAFE.setMemory(null, entriesAddress, buffer.capacity(), (byte) 0);
```

Comment:
For some interfaces `DirectUtils` has APIs like `getLong` or `putLong` and other interfaces are invoked referring to `UNSAFE` directly. Is it possible to move things like `setMemory(...)` to `DirectUtils` APIs?